mondrian/src/it/java/mondrian/test/SteelWheelsSchemaTest.java

```diff
@@ -2592,6 +2592,144 @@ public void testMondrian2411_3() throws Exception {
             + "{[Customer_DimUsage.Customers Hierarchy].[1 rue Alsace-Lorraine].[Roulet]}\n"
             + "Row #0: 1,701.95\n");
     }
+
+    /**
+     * this tests the fix for
+     * http://jira.pentaho.com/browse/Mondrian-2652
+     */
+    public void testMondrian2652() {
```

Comment:
Tests based on SteelWheels need to check if the test can run before it does. See the first few lines in test "testMeasures" above for an example.

Reply:
Great, that was the problem!